requirements.txt

```diff
@@ -1 +1,2 @@
 numpy>=1.17
+typing>=3.7
```

Comment:
Is typing not standard library? Does it need to be in requirements as it should come bundled with Python? Please correct me if I'm wrong

Reply:
Ignore this I was wrong